guides/05_chatbots/04_agents-and-tool-usage.md

```diff
@@ -0,0 +1,155 @@
+# Building a UI for an LLM Agent
+
+Tags: LLM, AGENTS, CHAT
+Related spaces: https://huggingface.co/spaces/gradio/agent_chatbot, https://huggingface.co/spaces/gradio/langchain-agent
+
+The Gradio Chatbot can natively display intermediate thoughts and tool usage. This makes it perfect for creating UIs for LLM agents. This guide will show you how. Before we begin, familiarize yourself with the `messages` chatbot data format documented in this [guide](/guides/05_chatbots/messages-format).
```

Comment:
This is what's currently breaking the build. Guide urls are just /guides/<name> there is no folder (chatbots in this case), and no numerical prefix. But also you're linking to a guide that doesn't exist on the release yet (because there hasn't been a release yet) so you need to link to the main url until then. The right way is to use a relative url so it finds the guide if it's on main or pip.  

```suggestion
The Gradio Chatbot can natively display intermediate thoughts and tool usage. This makes it perfect for creating UIs for LLM agents. This guide will show you how. Before we begin, familiarize yourself with the `messages` chatbot data format documented in this [guide](./messages-format).
```

Reply:
Done - thanks @aliabd !